examples/aldp.py

```diff
@@ -437,11 +486,13 @@ def ais_target_log_prob(x):
             x, log_w, log_q_old, indices = x.to(device), log_w.to(device), \
                                            log_q_old.to(device), indices.to(device)
             log_q_x = model.flow.log_prob(x)
-            # adjustment to account for change to theta since sample was last added/adjusted
-            w_adjust = torch.exp(log_q_old - log_q_x).detach()
-            w_adjust = torch.clip(w_adjust, max=rb_config['max_adjust_w_clip'])
-            # manually calculate the new form of the loss
+            # Adjustment to account for change to theta since sample was last added/adjusted
```

Comment:
Please check if this is implemented correctly, @lollcat.

Reply:
I ran a test on the GMM problem where I removed the w_adjust clipping and learning was still stable. So it may be worth adding an option here not to clip.